project/profiles/capacity4more/modules/c4m/og/c4m_og_menu/c4m_og_menu.features.menu_custom.inc

```diff
@@ -1,25 +0,0 @@
-<?php
-/**
- * @file
- * c4m_og_menu.features.menu_custom.inc
- */
-
-/**
- * Implements hook_menu_default_menu_custom().
- */
-function c4m_og_menu_menu_default_menu_custom() {
```

Comment:
Don't we need this anymore? The menu is appearing on the group dashboard pages, but it no longer appears in the menu overview for example.

<img width="801" alt="screen shot 2016-09-08 at 09 04 31" src="https://cloud.githubusercontent.com/assets/789034/18340096/57df214c-75a3-11e6-8f94-942534f57783.png">